File: C/armstrong.c
//check for armstrong number
#include <stdio.h>//success.....
int main(){//working....but only for 3 digit numbers
	printf("enter a number to check if its an armstrong number or not\n");
	int orinum;
	scanf("%d",&orinum);
	int k=0,d;
	int num=orinum;
	while(num!=0){
       d= num % 10;
       k+=(d*d*d);
       num=(num/10);
	}
	if(orinum==k){
		printf("%d is an armstrong number\n",orinum );
	}
	else{
		printf("NOT armstrong number\n");
	}
}

File: java/armstrong.java
import java.util.*;
public class armstrong
{
	public static void main(String sap[])//working...but only for 3 digit numbers...
	{
		String user;//success..........
		int len,num,i,ki,tri,next=0;
		Scanner nrt=new Scanner(System.in);
		System.out.println("enter a number...");
        user=nrt.nextLine();
        len=user.length();
        num=Integer.parseInt(user);
        for (i=0;i<len ;i++ ) {
        	char ch=user.charAt(i);
            ki=Integer.parseInt(""+ch);
            tri=(ki*ki*ki);
            next+=tri;
        }
        if (next==num) {
        	System.out.println("it is an armstrong number..");
        }
        else{
        	System.out.println("it is not armstrong number...");
        }
	}
}